conjurapi/client_test.go

```diff
@@ -32,32 +33,32 @@ func TestClient_GetConfig(t *testing.T) {
 			config: expectedConfig,
 		}
 
-		So(client.GetConfig(), ShouldResemble, expectedConfig)
+		assert.EqualValues(t, client.GetConfig(), expectedConfig)
```

Comment:
Same question as on another PR.... does `assert.EqualValues` work on maps? Wouldn't it be safer / more consistent to use `assert.True(t, reflect.DeepEqual(client.GetConfig(), expectedConfig)` ?

Reply:
I may be reading this wrong, but it looks like `assert.EqualValues` actually calls `reflect.DeepEqual` for you. Check out https://github.com/stretchr/testify/blob/6990a05d54c2287be8787a411d77815643347339/assert/assertions.go#L65

Reply:
NICE! Very good to know this.